lib/application.ex

```diff
@@ -205,6 +207,9 @@ defmodule Commanded.Application do
         Supervisor.stop(pid, :normal, timeout)
       end
 
+      @doc """
+      TODO: Does this function need to be documented?
+      """
```

Comment:
@slashdotdash what was the intention behind this function? I see it in my documentation behind my App module so I would like to improve the typespec and whatnot because of it.

Reply:
The documentation can be copied from the similar function defined in [`lib/commanded.ex`](https://github.com/commanded/commanded/blob/master/lib/commanded.ex#L43).